clang/lib/Lex/InitHeaderSearch.cpp

```diff
@@ -346,6 +346,7 @@ void InitHeaderSearch::AddDefaultIncludePaths(
         AddPath("/System/DriverKit/System/Library/Frameworks", System, true);
```

Comment:
Should we include SubFrameworks for DriverKit too, for consistency? Also, why's there an explicit /System/DriverKit here, didn't we generalize support for the system prefix?

Reply:
I'd rather discuss DriverKit support separately, so I will merge this pr now. 
> Also, why's there an explicit /System/DriverKit here, didn't we generalize support for the system prefix?
I don't know, but it does not look generalized to me even downstream. perhaps a question for @yln ?

Reply:
Followup in: https://github.com/llvm/llvm-project/pull/118543